package.json

```diff
@@ -10,6 +10,7 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "dependencies": {
-    "foreman": "^3.0.1"
+    "foreman": "^3.0.1",
+    "react-router-dom": "^6.21.3"
```

Comment:
This module is specific to the client, and should not be in the root package.json.

To install modules into a specific workspace, you can either `cd client` and install it there, or run `npm install <package> -w client`